polkadot/node/network/protocol/src/peer_set.rs

```diff
@@ -218,6 +229,9 @@ pub enum ValidationVersion {
 	V1 = 1,
 	/// The second version.
 	V2 = 2,
+	/// The staging version to gather changes
+	/// that before the release become v3.
+	VStaging = 3,
```

Comment:
This should be V3.

Reply:
See above, for why I decided to be staging instead of V3 to collect more changes before activating it.